packages/core/src/Avatar.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Flex from './Flex'
+import Heading from './Heading'
+import Text from './Text'
+import { User } from 'pcln-icons'
+
+const StyledImage = styled(Flex)`
+  border-radius: 50%;
+  width: 40px;
+  height: 40px;
```

Comment:
Would it make sense for the size to be configurable? One use case might be a small avatar in a menu bar and a large one on a profile page.